cdap-ui/app/features/overview/controllers/overview-ctrl.js

```diff
@@ -73,4 +73,27 @@ function ($scope, MyDataSource, $state, myLocalStorage, MY_CONFIG) {
     }
   });
 
+  $scope.wdgts = [];
+  var errLogMetric = 'system.log.error';
+  // type field is overridden by what is rendered in view because we do not use widget.getPartial()
+  $scope.wdgts.push(new Widget({title: 'Errors last minute - System', type: 'line', isLive: true,
```

Comment:
So now making a chart live becomes a property of widget instead of the chart? If I use epoch-line-chart directive somewhere I still need to use widget to make it a live one?